docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -74,101 +74,230 @@ ml_train_model_construct(&model);
 // Compile model. This freezes model and afterwards the model cannot be modified.
 ml_train_model_compile(model, "loss=cross", "batch_size=16", NULL);
 // run model
-ml_train_model_run(model, "epochs=2", "save_path=model.bin", NULL);
+ml_train_model_run(model, "epochs=2", NULL);
+
+/// Save, loading in below format supported since Tizen 6.5.
+ml_train_model_save(model, "model_weights.bin", ML_TRAIN_MODEL_FORMAT_BIN);
+ml_train_model_save(model, "model.ini", ML_TRAIN_MODEL_FORMAT_INI);
+ml_train_model_save(model, "model_and_weights.ini", ML_TRAIN_MODEL_FORMAT_INI_WITH_BIN);
 
 // destroy after use
 ml_train_model_destroy(model);
 ```
 
 A number of properties can be set at `ml_train_model_compile()` and `ml_train_model_run()` phase:
 
-Function | Key | Value | Description
--------- | --- | ----- | -----------
-`ml_train_model_compile()` | loss      | (categorical)| Loss function to be set
-&#xfeff;                 |           | cross        | cross-entropy loss
-&#xfeff;                 |           | mse          | mean squared error
-`ml_train_model_run()`     | epochs    | (integer)    | Determines epochs for the model
-&#xfeff;                 | save_path | (file_path)  | Model path to save and read parameters
+| Function                   | Key       | Value         | Description                            |
+| -------------------------- | --------- | ------------- | -------------------------------------- |
+| `ml_train_model_run()`     | epochs    | (integer)     | Determines epochs for the model        |
```

Comment:
why is the value written in brackets?
(integer) and  (file_path)

Reply:
It should be (integer) and (string) <- updated.

The value in bracket means it's a family of type not the actual value.

IMO it's (integer, boolean, string, floats) are quite common so wouldn't need additional explanation